torchtext/datasets/__init__.py

```diff
@@ -10,7 +10,7 @@
     AG_NEWS, SogouNews, DBpedia, YelpReviewPolarity, \
     YelpReviewFull, YahooAnswers, \
     AmazonReviewPolarity, AmazonReviewFull
-
+from .unsupervised_learning import EnWik9
```

Comment:
nit: same about import

Reply:
confused. This is grouped together with other `import`.